application/src/test/java/org/thingsboard/server/controller/BaseDashboardControllerTest.java

```diff
@@ -495,4 +477,28 @@ public void testAssignDashboardToEdge() throws Exception {
         Assert.assertEquals(0, pageData.getData().size());
     }
 
+    @Test
+    public void testDeleteDashboardWithRelationsOk() throws Exception {
+        DashboardId dashboardId = createDashboard().getId();
+        tesEntityDaoWithRelationsOk(savedTenant.getId(), dashboardId, "/api/dashboard/" + dashboardId);
+    }
+
+    @Test
+    public void testDeleteDashboardWithRelationsTransactionalException() throws Exception {
+        DashboardId dashboardId = createDashboard().getId();
+        tesEntityDaoWithRelationsTransactionalException(dashboardDao, savedTenant.getId(), dashboardId, "/api/dashboard/" + dashboardId);
+    }
+
+    private Dashboard createDashboard(String title) {
+        Dashboard dashboard = new Dashboard();
+        dashboard.setTitle(title);
+        return doPost("/api/dashboard", dashboard, Dashboard.class);
+    }
+
+    private Dashboard createDashboard() {
```

Comment:
remove. code is almost the same. it's enough to call `createDashboard(String title)` with the name.

Reply:
Ok